core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/CodeGenerationTests.kt

```diff
@@ -305,6 +309,38 @@ class CodeGenerationTests : BaseTest() {
         }
     }
 
+    @Test
+    fun `data classses`() {
+        val code = typed.groupBy { name }.toDataFrame().generateDataClasses()
+
+        code shouldBe """
+            @DataSchema
+            data class Person1(
+                val age: Int,
+                val city: String?,
+                val name: String,
+                val weight: Int?
+            )
+
+            @DataSchema
+            data class Person(
+                val group: List<Person1>,
+                val name: String
+            )
+        """.trimIndent().toCodeString()
+    }
+
+    @Test
+    fun `normalize name`() {
```

Comment:
renameTo -> `check name normalization for generated data classes`

Reply:
done, thank you!